geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -104,11 +126,11 @@ where
     compute_voronoi_components_from_delaunay(&triangles, clipping_mask, slope_threshold)
 }
 
-/// Compute the Voronoi Diagram from Delaunay Triangles.
-/// The Voronoi Diagram is a [dual graph](https://en.wikipedia.org/wiki/Dual_graph)
-/// of the [Delaunay Triangulation](https://en.wikipedia.org/wiki/Delaunay_triangulation)
-/// and thus the Voronoi Diagram can be created from the Delaunay Triangulation.
-pub fn compute_voronoi_components_from_delaunay<T: GeoFloat>(
+/// Compute the Voronoi diagram from Delaunay triangles.
+/// The Voronoi diagram is a [dual graph](https://en.wikipedia.org/wiki/Dual_graph)
+/// of the [Delaunay triangulation](https://en.wikipedia.org/wiki/Delaunay_triangulation)
+/// and thus the Voronoi diagram can be created from the Delaunay triangulation.
+fn compute_voronoi_components_from_delaunay<T: GeoFloat>(
     triangles: &[Triangle<T>],
     clipping_mask: Option<&Polygon<T>>,
```

Comment:
mention the clipping_mask and slope_threshold in the docstring, mentioning that they default to `DEFAULT_CLIPPING_MASK_EXPANSION` and `DEFAULT_SLOPE_THRESHOLD` ?

Reply:
actually :-1: this isnt `pub` so not worth duplicating description of those two args here.